### DISCUSSION TITLE
forum-session-HTML

### DISCUSSION THREAD

**Discussion by giscus** [arXiv/accessibility2024#131] [General]
# forum-session-HTML

Join four open-source software luminaries from the NVDA screenreader, the LaTeXML converter, and arXiv to take a peek under the hood of how HTML papers on arXiv are made, what is working well, and where there is room for improvement.

https://accessibility2024.arxiv.org/forum-session-HTML

**Comment by SBBCornell** [comment]
Welcome everyone! Please share thoughts, questions, and resources of interest to the community.

**Comment by Wednesday-42** [comment]
I found the panel very informative! One panelist made a remark near the end that really made me aware of the assumptions I've been making when I just blindly pass my documents through LaTeXML to create HTML output.

Where can I find a list of standard LaTeX symbols that are supported by LaTeXML? Is there a list of those that still have yet to be implemented? I was really surprised to learn that \aleph isn't supported right now, since that's in "vanilla" LaTeX and widely used for cardinalities of sets, so I'm sure there must be a lot of other stuff I've been assuming is fine that isn't!

**Reply by dginev** [reply]
Thank you for the question. \aleph is indeed already supported by LaTeXML. The warning during the session was about screen-reading software not having vocalizations for the full range of Unicode characters, and suggesting the aleph entry “ℵ” (U+2135) is a hard case. I suspect this would also be challenging in Braille displays.

Tony's suggestion was to stick to Latin and Greek alphabets whenever possible. But naturally, requesting broader coverage of Unicode from accessibility tools is also important.

As to LaTeXML documentation, we don't support a static list of supported macros, but you can use github search to discover if a macro is already present. Here is the result for \aleph:
https://github.com/search?q=repo%3Abrucemiller%2FLaTeXML%20%5Caleph&type=code

Since our goal with LaTeXML is to cover the full breadth of arXiv, I think it is more fruitful to report missing macros that you'd really want added - after testing during the arXiv submission, or locally.

**Reply by xworld21** [reply]
@dginev the DLMF adds a 'title' attribute to certain macro (for instance, the capital gamma in https://dlmf.nist.gov/5.2 has title 'gamma function'). Is that something that can be done easily from LaTeXML?

**Reply by dginev** [reply]
@xworld21 Indeed. DLMF is generated via LaTeXML with some post-processing customization, and a LaTeX authoring toolbox of its own "semantic macros".

The two pieces that need to be added on that front:
 - Publishing the macro dialect used in DLMF - or a generalization of it - for public use.
 - Adopting a more accessibility-oriented attribute dialect, such as the upcoming MathML 4 Intent specification.

There are some plans in that direction on the LaTeXML side, but you can always send some additional encouragement, and especially in Bruce Miller's general direction.

**Comment by Wednesday-42** [comment]
Oh! And another question. One panelist mentioned how making documents screen-reader accessible also makes them easier for companies to scrape for genAI training sets.  Is there any meta tag we can add to a LaTeXML document to discourage that, along the lines of noindex?

**Reply by dginev** [reply]
To my knowledge, there is no such tag. But this may also require additional regulatory steps, as I suspect there is no universally accepted mechanism to enforce this from the web technology stack itself.

**Comment by artunit** [comment]
This was an excellent session. There was a brief discussion at the end in the Q&A about Theses and Dissertation submissions at universities. It would seem to be a good point to ensure accessible equations but PDF renderings as shown in the demo for this session seem to be the norm. Has anyone come across accessibility guidelines for graduate work that explicitly require accessible math?

**Reply by SBBCornell** [reply]
I wanted to share the responses from two attendees at the live event here:

1. "In my experience they are not. A recent conversation about "_what to do about the dissertation archive"_ and going forwards those to be added... no plan."
2. "My experience is the same. They are usually PDF generated with LaTeX, and the formulae are not accessible. Changing the standard from PDF to HTML (or some equivalent) is really the way."
